sdk/eventhub/eventhubs-checkpointstore-blob/README.md

```diff
@@ -76,7 +76,12 @@ await containerClient.create(); // This can be skipped if the container already
 const partitionManager =  new BlobPartitionManager(containerClient);
 
 SamplePartitionProcessor extends PartitionProcessor  {
-   // add code here to override the processevents method to checkpoint the last event in the batch
+   processEvents(events, partitionContext) {
```

Comment:
Sorry, minor, but can you make this `async`? That way it matches the expected signature